pangeo-notebook/packages.txt

```diff
@@ -135,22 +135,22 @@ freeglut-3.2.1
 freetype-2.10.4
 freexl-1.0.6
 fribidi-1.0.10
-fsspec-2021.8.1
+fsspec-2021.9.0
 fsspec-reference-maker-0.0.3
 future-0.18.2
 gcc_impl_linux-64-11.2.0
 gcc_linux-64-11.2.0
-gcsfs-2021.8.1
-gdal-3.3.1
+gcsfs-2021.9.0
+gdal-3.3.2
 gdk-pixbuf-2.42.6
 geocube-0.0.17
 geographiclib-1.52
 geopandas-0.9.0
 geopandas-base-0.9.0
 geopy-2.2.0
 geos-3.9.1
-geotiff-1.6.0
-geoviews-core-1.9.1
+geotiff-1.7.0
+geoviews-core-1.8.2
```

Comment:
not sure why `geoviews-core` dropped from 1.9.1 to 1.8.2... also not sure if we need to keep using `geoviews-core` or just geoviews...